app/lib/rails_i18n_manager/google_translate.rb

```diff
@@ -13,7 +14,7 @@ def self.translate(text, from:, to:)
         return nil
       end
 
-      EasyTranslate.translate(text, from: from, to: to, key: api_key)
+      translation = EasyTranslate.translate(text, from: from, to: to, key: api_key)
```

Comment:
This looks fine for this PR. Though I wonder why we have some complexity in the statements below. They probably need to be refractored a bit.

```ruby
if translation
  str = translation.to_s
  
  if str.present?
```